Mathlib/Data/Nat/Pairing.lean

```diff
@@ -132,8 +132,8 @@ theorem pair_lt_pair_left {a₁ a₂} (b) (h : a₁ < a₂) : pair a₁ b < pair
   · simp at h₁
     simp only [not_lt_of_gt (lt_of_le_of_lt h₁ h), ite_false]
     apply add_lt_add
-    exact Nat.mul_self_lt_mul_self h
-    apply Nat.add_lt_add_right; assumption
+    · exact Nat.mul_self_lt_mul_self h
```

Comment:
This is adding cdots (but looks like a good change regardless).

Reply:
You are right: I messed up with git and pulled in a change from the other PR that was *adding* the cdots.